src/services/web-monitoring-db.ts

```diff
@@ -72,6 +72,18 @@ export default class WebMonitoringDb {
             .then(data => parsePage(data.data));
     }
 
+    getPagesByDomains (domains: string[]): Promise<Page[]> {
+        const daysAgo = 3;
+        const dateEarlier = new Date(new Date().setDate(new Date().getDate() - daysAgo)).toISOString();
```

Comment:
This is an good start, but eventually we should probably be checking defined timeframes. Technically, the timeframe an analyst is looking at isn’t the last 3 days, it’s a specific 3-day period that may have ended this morning, yesterday, or even the day before.

I’m thinking we’ll want to define that somehow in another spreadsheet, since @mayaad and @trinberg occasionally adjust the analysis window. The right approach for this might be something you should spend some time talking with them about.